lib/1.4/dml-builtins.dml

```diff
@@ -659,19 +659,17 @@ as parent.
 The `group` template contains no particular methods or parameters other
 than what is inherited from the [`object`](#object) template.
 
-In a group hierarchy directly below a `port` or `device` object, none of the
-group object may be named `bank` or `port`; this is to avoid name clashes.
+You may not declare any `bank`, `port` or `subdevice` underneath any group
+named "bank" or "port". This is to avoid namespace clashes in Simics.
 */
 template group is object {
     param objtype = "group";
     param _is_simics_object = false;
     param log_group = undefined;
-    #if ((_nongroup_parent.objtype == "subdevice"
-          || _nongroup_parent.objtype == "device")
-         && (this.name == "port" || this.name == "bank")) {
-         error "Cannot name a group 'port' or 'bank' in a location where"
-             + " port or bank objects are allowed";
-    }
+    param _simics_namespace_clash =
```

Comment:
It's initially struck me as concerning that we add a new param to our most lightweight object. But I doubt it can be a problem, should be the same kind of expensive as the `#if` we remove.